dags/ods/kktix_ticket_orders/udfs/klaviyo_mailer.py

```diff
@@ -0,0 +1,163 @@
+"""Send a mail via Klaviyo
+
+Requirements:
+
+1. Create a [Klaviyo List](https://www.klaviyo.com/lists)
+
+2. Create a template [campaign](https://www.klaviyo.com/campaigns) and set the previous List as target recipients list
+
+"""
+import requests
+import tenacity
+from datetime import datetime
+from typing import List
+from airflow.hooks.http_hook import HttpHook
+from airflow.models import Variable
+
+
+SCHEDULE_INTERVAL_SECONDS: int = 300
+RETRY_ARGS = dict(
+    wait=tenacity.wait_none(),
+    stop=tenacity.stop_after_attempt(3),
+    retry=tenacity.retry_if_exception_type(requests.exceptions.ConnectionError),
+)
+
+
+def main(
+    list_id: str,
+    campaign_id: str,
+    campaign_name: str,
+    datas: List[dict],
```

Comment:
@henry410213028 only 1 question, the data here are hashed emails, would that be a problem?